src/sentry/static/sentry/app/views/organizationIntegrations/addIntegration.jsx

```diff
@@ -46,12 +46,12 @@ export default class AddIntegration extends React.Component {
     return {left, top};
   }
 
-  openDialog = urlParams => {
+  openDialog = event => {
     const name = 'sentryAddIntegration';
     const {url, width, height} = this.props.provider.setupDialog;
     const {left, top} = this.computeCenteredWindow(width, height);
 
-    const query = {...urlParams};
+    const query = {};
```

Comment:
This is going to break the install from github flow:

https://github.com/getsentry/sentry/blob/ef62900b86cb08cc9abdb5b8f49a1c770f86be3d/src/sentry/static/sentry/app/views/integrationInstallation.jsx#L85

Can we change the click handler to take a wrapper function that discards the event instead?